Makefile

```diff
@@ -110,3 +110,7 @@ markdownlint:
 
 markdownlint-fix:
 	docker run -v $(CURDIR):/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:latest  "**/*.md" --config "/workdir/docs/markdownlint-rules.yaml" --fix --ignore "/workdir/CHANGELOG.md"
+
+yamllint:
+	@docker run --rm -t -v $(PWD):/data cytopia/yamllint:latest .github docs
```

Comment:
please use version `alpine-1-0.14` instead of `latest` so that we have a pinned and reproducible version.
Ideally, you follow the scheme we use for example in the docs/Makefile where we have the version in a separate variable at the top of the file and then reference it.
Then you can add a renovate annotation comment so that we get automatic updates for the docker image version.
Check [here](https://github.com/keptn/lifecycle-toolkit/blob/main/docs/Makefile#L3-L4) for an example.